package.json

```diff
@@ -91,11 +91,11 @@
     "tap": "^11.0.0",
     "then-sleep": "^1.0.1",
     "typescript": "^2.6.2",
+    "@types/node": "^8.5.2",
+    "@types/pino": "^4.7.0",
```

Comment:
can you restore these?

Reply:
Yes, was it intentional?

Reply:
Why we need this in dependencies?

Reply:
Yes it was intentional and discussed somewhere else, seems the best approach to support typescript.

Reply:
As far I know the user is responsible to include the correct definition files. For development it should be enough to install them as dev dependencies. Could you point the issue?